tasmota/tasmota.ino

```diff
@@ -223,7 +223,7 @@ bool tasconsole_serial = false;
 
 #if ARDUINO_USB_MODE
 //#warning **** TasConsole ARDUINO_USB_MODE ****
-TASCONSOLE TasConsole{USBSerial};           // ESP32C3/C6/S3 embedded USB using JTAG interface
+TASCONSOLE TasConsole{HWCDCSerial};         // ESP32C3/C6/S3 embedded USB using JTAG interface
```

Comment:
Where does this change come from?

Reply:
Latest Arduino 3.0.0 build (Platform 2023.10.12) needs that change.
Commit https://github.com/tasmota/arduino-esp32/commit/9dd9bde682c87055dac93f6abf3fed9dafaa64d9